priv/schemas/v0-1-0/content-element.schema.json

```diff
@@ -889,8 +877,25 @@
         "$ref": "#"
       }
     },
+    "figure": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "const": "figure"
+        },
+        "title": {
+          "type": "string"
+        },
+        "children": {
+          "type": "array",
+          "items": {
+            "$ref": "#/$defs/semantic-element-content"
+          }
+        }
+      }
+    },
 
-    "meaning-content": {
+    "semantic-element-content": {
```

Comment:
Took meaning-content from the definitions, generalized the name to semantic-content and plan on using that as the content type of the children in several of these new semantic elements.